### PR TITLE
The Guild - Rebranding in open source

### PR DESCRIPTION
**Pull request by Urigo**

### CODE REVIEW

File: website/pages/blog/rebranding-in-open-source.mdx [hunk 1]
```diff
@@ -0,0 +1,181 @@
+---
+title: The Guild - Rebranding in open source
+tags: [open-source, branding, marketing]
+authors: [urigo]
```
Comment (kamilkisiela) [inline website/pages/blog/rebranding-in-open-source.mdx:4]:
```suggestion
authors: [uri]
```

File: website/pages/blog/rebranding-in-open-source.mdx [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+title: The Guild - Rebranding in open source
+tags: [open-source, branding, marketing]
+authors: [uri]
+date: 2024-02-20
+description: The Guild is doing rebranding - The open source way
+image: /blog-assets/graphql-yoga-nest-v9/cover.png
+thumbnail: /blog-assets/graphql-yoga-nest-v9/thumbnail.png
+---
+
+## tldr
```
Comment (saihaj) [inline website/pages/blog/rebranding-in-open-source.mdx:11]:
```suggestion
## TL;DR
```

File: website/pages/blog/rebranding-in-open-source.mdx [hunk 3]
```diff
@@ -0,0 +1,181 @@
+---
+title: The Guild - Rebranding in open source
+tags: [open-source, branding, marketing]
+authors: [uri]
+date: 2024-02-20
+description: The Guild is doing rebranding - The open source way
+image: /blog-assets/graphql-yoga-nest-v9/cover.png
+thumbnail: /blog-assets/graphql-yoga-nest-v9/thumbnail.png
```
Comment (saihaj) [inline website/pages/blog/rebranding-in-open-source.mdx:8]: <insert new thumbnail>

File: website/pages/blog/rebranding-in-open-source.mdx [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+  company, structured around the most solid solutions in open source that could be relied upon by
+  users and tool builders
+- **More exposure** - More developers to be aware of the available products we offer and why they
+  are the best in the industry (if they are not, we want to understand why as part of that process,
+  we want to share that as well in public). That includes the GraphQL ecosystem but also the wider
+  API ecosystem
+
+## The process
+
+Here is the process we’ll go through, led by North Star (they've also wrote about the process
```
Comment (saihaj) [inline website/pages/blog/rebranding-in-open-source.mdx:78]:
```suggestion
Here is the process we’ll go through, led by [North Star](https://www.northstar.cx) (they've also wrote about the process
```

File: website/pages/blog/rebranding-in-open-source.mdx [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+- We’ve decided its time for The Guild to go through a professional process of branding and
+  marketing
+- We’ve hired experts to help us in that process - North Star
+- Like everything we do at The Guild - we’ve decided to take the whole process open source and share
+  with our community everything in the process, including our wins and painful points
+
+## The Guild today
+
+The Guild are a group of developers, creating open source, developer experience and infrastructure
+tools for managing data and APIs.
```
Comment (saihaj) [inline website/pages/blog/rebranding-in-open-source.mdx:27]:
```suggestion
## The Guild Today

We are a group of developers, creating open source, developer experience and infrastructure
tools for managing data and APIs.
```

File: website/pages/blog/rebranding-in-open-source.mdx [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+  API ecosystem
+
+## The process
+
+Here is the process we’ll go through, led by North Star (they've also wrote about the process
+[here](https://www.northstar.cx/blog-detail/the-guild-in-the-wind-of-change-open-source-rebranding-in-public)):
+
+### First phase — Brand Audit
+
+During that process, NorthStar will analyse the current state of The Guild
```
Comment (saihaj) [inline website/pages/blog/rebranding-in-open-source.mdx:83]:
```suggestion
During that process, [North Star](https://www.northstar.cx) will analyse the current state of The Guild
```

File: website/pages/blog/rebranding-in-open-source.mdx [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+
+They will learn about:
+
+- How we communicate
+- Our current brand and identity
+- Our value propositions
+- The overall ecosystem.
```
Comment (saihaj) [inline website/pages/blog/rebranding-in-open-source.mdx:90]:
```suggestion
- The overall ecosystem
```

File: website/pages/blog/rebranding-in-open-source.mdx [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+The Guild was created to be a place where solutions that are sustainable and for the long term are
+being built.
+
+One of the best ways to accomplish that is to be fully open source. It means that no matter what
+happens, including anything with the company itself — the solutions and the code are still
+available, out there and anyone could use and build upon them.
+
+As we’ve successfully managed to create that environment, it also came with a great deal of freedom.
+
+For example, we could have our most successful product - GraphQL Hive - to be fully open source!
```
Comment (saihaj) [inline website/pages/blog/rebranding-in-open-source.mdx:165]:
```suggestion
For example, we could have our most successful product - [GraphQL Hive](https://the-guild.dev/graphql/hive) - to be fully open source!
```

File: website/pages/blog/rebranding-in-open-source.mdx [hunk 9]
```diff
@@ -0,0 +1,181 @@
...
+  sustainable way, like we have successfully managed to do
+- We’ve decided its time for The Guild to go through a professional process of branding and
+  marketing
+- We’ve hired experts to help us in that process - North Star
+- Like everything we do at The Guild - we’ve decided to take the whole process open source and share
+  with our community everything in the process, including our wins and painful points
+
+## The Guild today
+
+The Guild are a group of developers, creating open source, developer experience and infrastructure
```
Comment (YassinEldeeb) [inline website/pages/blog/rebranding-in-open-source.mdx:26]:
```suggestion
The Guild is a group of developers, creating open source, developer experience and infrastructure
```

File: website/pages/blog/rebranding-in-open-source.mdx [hunk 10]
```diff
@@ -0,0 +1,181 @@
...
+instead of short term, trend and marketing based software foundations.
+
+We’ve had a unique vision and a way to achieve it and we’ve manage to be successful at it.
+
+At this stage, we feel that more people should be aware of what we do.
+
+Some as users of our tools and some as other open source developers and solution providers, to share
+our tools and learnings with the wider community.
+
+We never done any marketing and we think this is a good time to start - At this stage, marketing to
```
Comment (YassinEldeeb) [inline website/pages/blog/rebranding-in-open-source.mdx:50]:
```suggestion
We have never done any marketing and we think this is a good time to start - At this stage, marketing to
```

File: website/pages/blog/rebranding-in-open-source.mdx [hunk 11]
```diff
@@ -0,0 +1,181 @@
...
+## Where we want to be, our goals from the process
+
+As we go through the process, we have to ask ourselves what do we actually want to get.
+
+Our current goals as we see them today:
+
+- **A clear offering** - We offer many products, spread across many places across the stack. It is
+  currently very hard for someone new to The Guild’s ecosystem to understand what each tool does and
+  how it could benefit them
+- A philosophy of development - We’ve managed to create a philosophy around how we work, in order to
```
Comment (YassinEldeeb) [inline website/pages/blog/rebranding-in-open-source.mdx:62]:
```suggestion
- **A philosophy of development** - We’ve managed to create a philosophy around how we work, in order to
```

File: website/pages/blog/rebranding-in-open-source.mdx [hunk 12]
```diff
@@ -0,0 +1,181 @@
...
+- **Hiring** - We want to give ambitious open source developers a career path that they are not
+  currently getting in other companies. Maximizing their individual potential is the core reason of
+  why we’ve created The Guild in the first place. We want to let developers know that The Guild is
+  more than a GraphQL provider and it’s also more than an API provider — it’s the open source
+  company, structured around the most solid solutions in open source that could be relied upon by
+  users and tool builders
+- **More exposure** - More developers to be aware of the available products we offer and why they
+  are the best in the industry (if they are not, we want to understand why as part of that process,
+  we want to share that as well in public). That includes the GraphQL ecosystem but also the wider
+  API ecosystem
```
Comment (YassinEldeeb) [inline website/pages/blog/rebranding-in-open-source.mdx:74]:
```suggestion
- **Clear offering:** We offer many products, spread across many places across the stack. It is
  currently very hard for someone new to The Guild’s ecosystem to understand what each tool does and
  how it could benefit them
- **Philosophy of development:** We’ve managed to create a philosophy around how we work, in order to
  create sustainable, reliable, quality open source infrastructure. We want to share these ideas
  with our users and other open source developers
- **Hiring:** We want to give ambitious open source developers a career path that they are not
  currently getting in other companies. Maximizing their individual potential is the core reason of
  why we’ve created The Guild in the first place. We want to let developers know that The Guild is
  more than a GraphQL provider and it’s also more than an API provider — it’s the open source
  company, structured around the most solid solutions in open source that could be relied upon by
  users and tool builders
- **More exposure:** More developers to be aware of the available products we offer and why they
  are the best in the industry (if they are not, we want to understand why as part of that process,
  we want to share that as well in public). That includes the GraphQL ecosystem but also the wider
  API ecosystem
```

File: website/pages/blog/rebranding-in-open-source.mdx [hunk 13]
```diff
@@ -0,0 +1,181 @@
...
+communication strategy and guidelines and any other product or marketing related work we’ll execute
+on.
+
+Usually these processes are not being done in the open, even at open source companies.
+
+We’ll need to use or create tools and ways that help us collaborate on the process in that unique
+way. We hope others could benefit from that as well and maybe make it easier for companies to go
+through these types of processes in the open.
+
+## Why are we sharing all of it in the open
```
Comment (YassinEldeeb) [inline website/pages/blog/rebranding-in-open-source.mdx:154]:
```suggestion
## Why are we sharing all of it in the open?
```

File: website/pages/blog/rebranding-in-open-source.mdx [hunk 14]
```diff
@@ -0,0 +1,181 @@
...
+As we’ve successfully managed to create that environment, it also came with a great deal of freedom.
+
+For example, we could have our most successful product - GraphQL Hive - to be fully open source!
+While still being extremely successful SaaS business.
+
+That freedom drives us to work on things in the best way possible and we believe that sharing the
+marketing and branding process with all of you, will help us achieve the best results.
+
+It would help others who want to go through similar process, it will give access to our community to
+influence and take part this important process and it would help other GraphQL vendors and push the
```
Comment (YassinEldeeb) [inline website/pages/blog/rebranding-in-open-source.mdx:172]:
```suggestion
influence and take part in this important process and it would help other GraphQL vendors and push the
```

File: website/pages/blog/rebranding-in-open-source.mdx (hunk 11 above)
Comment (enisdenjo) [inline website/pages/blog/rebranding-in-open-source.mdx:62]:
```suggestion
- **A philosophy of development** - We’ve managed to create a philosophy around how we work, in order to
```

**Review by saihaj — APPROVED** [review]
